src/controllers/user/me.rs

```diff
@@ -36,12 +36,27 @@ pub fn me(req: &mut dyn Request) -> CargoResult<Response> {
         ))
         .first::<(User, Option<bool>, Option<String>, bool)>(&*conn)?;
 
+    let owned_crates = crate_owners::table
+        .inner_join(crates::table)
+        .filter(crate_owners::owner_id.eq(user_id))
```

Comment:
I forgot to mention this in the instructions and the code doesn't make this very clear, but this query needs to also have a filter for the `crate_owners::owner_kind` being `OwnerKind::User` because records for teams owning crates are also stored in this table. Here's an example of a similar query: https://github.com/rust-lang/crates.io/blob/ea248e161b22dd0e7f132825333d0e4383d301c1/src/controllers/user/other.rs#L39

I'm starting to think we should split team and user ownership records into separate tables, but that's a totally separate issue 😅